src/components/Button.js

```diff
@@ -14,8 +14,8 @@ class Button extends React.Component {
   static propTypes = {
     'aria-label': PropTypes.string,
     actionText: PropTypes.string,
-    buttonRef: PropTypes.func,
```

Comment:
Renaming this prop would introduce a breaking change. Can you remove these changes to `Button.js` for now?